core/services/cable_guy/html/index.html

```diff
@@ -89,38 +89,38 @@
       // Fetch periodic API request
       this.startPeriodicRequest()
     },
-    updated: function () {
+    updated () {
```

Comment:
Aren't those spaces between function name and parenthesis do ugly?

Reply:
That's a rule that exists before this PR.

Reply:
I'm arguing that we should add here :smile:

Reply:
I'm ok with it, just want to know @Williangalvani reason for such thing.

Reply:
I'm just used to it. It also seems to be part some kind of [js standard](https://standardjs.com/rules.html). If it is on the internet it should be right, right?

Reply:
If it's the standard I'm ok with it.
Everything else was addressed, please approve the PR.

Reply:
Bizarre, but if it's the convention I'm also ok.